### PR TITLE
Create Postgres schema and be able to connect to database

### PR DESCRIPTION
**Pull request by ajfojas**
@howardyang25 You mind double checking my work? Thanks!

### CODE REVIEW

File: server/db/seedpsql.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+const dbpsql = require('./indexpsql.js');
+
+const tenMil = 10000000;
+
+const randomInt = function (max) {
+  let num = Math.ceil(Math.random() * max);
+  // all IDs start at 1 and not 0
+  if (num < 1) {
+    num = 1;
+  }
```
Comment (howardyang25) [inline server/db/seedpsql.js]: Is this conditional necessary? Wouldn't Math.ceil ensure that the lowest generated number is 1?

File: server/db/seedpsql.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      } else {
+        console.log('PHOTO INSERT SUCCESS');
+      }
+    });
+  }
+}
+
+console.log('seeder running');
+
+dbpsql.pool.end();
```
Comment (howardyang25) [inline server/db/seedpsql.js]: Unsure, but does ending the connection here ever result in an error for you? I've always had trouble finding a way to end the connection after all the async functions have finished running.

File: server/db/indexpsql.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+const { Pool } = require('pg');
+
+const pool = new Pool({
+  host: 'localhost',
+  user: 'student',
+  password: 'student',
+  database: 'gallerymodule',
+});
```
Comment (howardyang25) [inline server/db/indexpsql.js]: Are you able to connect without indicating a port?

File: cschema.cql [hunk 4]
```diff
@@ -8,9 +8,15 @@ CREATE KEYSPACE gallerymodule
 
 USE gallerymodule;
 
-CREATE TABLE listings (
-  id SERIAL PRIMARY KEY,
-  src VARCHAR(100) NOT NULL,
-  description VARCHAR(100) NOT NULL,
+CREATE TABLE images (
+  id            UUID PRIMARY KEY,
```
Comment (howardyang25) [inline cschema.cql]: Reasoning for using UUID? I don't know too much about it.

File: server/db/seedpsql.js (hunk 1 above)
Comment (ajfojas) [reply server/db/seedpsql.js]: I think you're right, thanks :)

File: server/db/seedpsql.js (hunk 2 above)
Comment (ajfojas) [reply server/db/seedpsql.js]: I've never run into troubles yet but I'll keep this in mind, thanks

File: server/db/indexpsql.js (hunk 3 above)
Comment (ajfojas) [reply server/db/indexpsql.js]: Yeah, I believe I only need the port number for my Express server. I've had problems explicitly defining a port number

File: cschema.cql (hunk 4 above)
Comment (ajfojas) [reply cschema.cql]: From what I've read, I believe it's to make the id unique

**Pull request merged**